NumLib/TimeStepping/Algorithms/TimeStepAlgorithm.h

```diff
@@ -85,6 +85,12 @@ class TimeStepAlgorithm
     /// Get a flag to indicate whether this algorithm needs to compute
     /// solution error. The default return value is false.
     virtual bool isSolutionErrorComputationNeeded() { return false; }
+
+    /// Add specific times
+    virtual void addSpecificTimes(std::vector<double> const& /*specific_times*/)
```

Comment:
Actually the general description would be here, and the overriding function in PID controller would provide additional info...

Reply:
Use \copydoc